src/modules/spreadsheet/spreadsheet.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
I deleted old `.js` files, so I don't want to try to fix this unused file. Anyway we will work on this file soon